LDY/Day5/음양 더하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(absolutes, signs) {
```

Comment:
너무 너무 완벽 그자체!!
저는 실무 관점에서 리뷰 해드릴게요 ^^

```suggestion
function solution(absolutes, signs) {
if (absolutes.length !== signs.length) {
    // 실제로는 Error 클래스를 커스텀하거나 다른 오픈소스 Error 라이브러리를 활용하기도 합니다.
    // ex: throw new InvalidInputError()
    throw new Error("입력 배열의 길이가 서로 다릅니다.");
  }
```

실제로는 모든 함수 별로 입력값이 정상적으로 들어왔는지 검증하는 것이 필요해요.
타입 검사라든지, 의도한 입력값인지 라든지 말이죠?

그러나 이러한 검증들이 많아지면 복잡해지기 때문에 이들을 자동적으로 처리해주는 라이브러리의 도움을 받는 편이에요.
혹은 자동으로 지원되는 언어를 사용한다던지요. (ex: 타입스크립트)

자바스크립트는 타입이 자유롭기 때문에 습득과 성장은 빠르지만, 서비스가 대규모로 갈 수록 유지보수가 어려워지는 단점이 명확해요.
때문에 실제 서비스나 대규모를 다룰때는 그에 필요한 라이브러리들의 도움을 필히 받아야합니다.
이해 안되면 10기 선배들께 여쭤보세요 ^^

지금은 그냥 아.. 이런게 있구나 하고 넘어가시고 프로젝트 개발할 때 적용해보세요.
코테에서는 이런 Error 검증은 시간 낭비이니 하지 마시면 안되구 :D

Reply:
안전하게 작성하는 방법에 자세한 피드백까지 설명해주셔서 정말 감사드립니다!!🙇🏻‍♂️🙇🏻‍♂️